### DISCUSSION TITLE
What is Comet's plan for the data lake ecosystem?

### DISCUSSION THREAD

**Discussion by dpengpeng** [apache/datafusion-comet#1018] [General]
The big data analytics platform has begun to evolve towards a lakehouse architecture, with the computational engine Spark starting to interface with data lake table formats, such as Delta Lake, Iceberg, and Hudi. What is the current progress of Comet in supporting data lake table formats, which formats does it plan to support, and is there an approximate release timeline?
Thanks.

**Comment by viirya** [comment]
We recognize the importance of these data formats in current data systems. Among the above formats, we are working on Iceberg support. I'd like to see Iceberg in next releases but it depends on actual development. For other formats, we currently don't have bandwidth. So we may not be able to support them soon.

**Comment by andygrove** [comment]
Here is the tracking issue for Iceberg support: https://github.com/apache/datafusion-comet/issues/1028
